prestep.F90

```diff
@@ -58,15 +58,13 @@ subroutine prestep(carma, cstate, rc)
           ! (i.e. supersaturated) and d_gc is negative. For better performance,
           ! substep the gas unless both of these are true. This might run into
           ! trouble if d_t is large and negative.
-!          if (d_gc(iz, igas) < 0._f) then
-          if ((d_gc(iz, igas) < 0._f) .and. ((supsatiold(iz, igas) > 0._f) &
```

Comment:
we may want to keep this two lines as comments here since people may want to use this one instead sometimes.

Reply:
Maybe, but it is dangerous to leave code around that isn't used and isn't used for a good reason (i.e. it crashes the model).

Reply:
make sense, agree with Chuck.

Reply:
is already done